### ISSUE TITLE
refactor and improve [simple request app] with crayons

### ISSUE THREAD

**Issue by Saif-Shines** [freshworks/marketplace-sample-apps#190]
- [ ] Find sample code for `simple request app` app in the repository and tell us(in comments) if you were able to find it under 30s.
- [ ] (Optional) Add [summary in JSDoc commenting standards](https://jsdoc.app/tags-summary.html) and [elaborate parameters](https://jsdoc.app/tags-param.html) if you understand the code.
- [ ] Lower case the contents inside the `screenshots` folder (and `Screenshots` folder itself). If there's no screenshots just add it if it's a frontend app or else move on to next task. 
- [ ] Update the `README.md` file so that it follows [the standard format](https://github.com/freshworks/marketplace-sample-apps/blob/master/.github/PULL_REQUEST_TEMPLATE/APP_README_TEMPLATE.md). Don't worry about filling in all the details. Just give it a shot.
- [ ] Update the `README.md` file of the same app so that images inside of `screenshots` folder are displayed on the app README.md file.
- [ ] If you find any `iparams_test_data.json` inside of `config/` directory. Delete it.
- [ ] Re organise all the files and folders in following structure. Make sure the `html`, `css` and `js` files have right references to `src`s.
```
    .
    ├── README.md                   A file for future self and friends to learn about app
    ├── app                         A folder to place files needed for app's frontend components
    │   ├── index.html              The Entry point for the user to use the app
    │   ├── scripts                 JavaScript to handle frontend components business logic
    │   │   └── app.js
    │   ├── styles                  A folder to place all the styles for the app
    │   │   ├── images              A folder for all the images
    │   │   │   └── icon.svg
    │   │   └── styles.css
    │   └── views                   A folder to place all the pages
    │       └── modal.html
    ├── config                      A folder to place configuration pages
    │   └── iparams.json            A JSON oriented way to construct a configuration page
    └── manifest.json               A JSON file holding meta data for app to run on platform

```
- [ ] Install [Freshworks CLI](https://community.developers.freshworks.com/t/what-are-the-prerequisites-to-install-the-freshworks-cli/234)
- [ ] Rewrite UI elements using [Crayons kit](https://crayons.freshworks.com/components/). For example, this app uses buttons built using css(either from script tag or css file). Why not make it simple with crayons?
- [ ] Locally run the app by getting a trial account for relevant Freshworks' product (*Hint*: See `manifest.json`). We will let you figure out how to do that by exploring the [developer documentation](https://developers.freshworks.com/). How much time did you take to figure out? 2 min?
- [ ] Take a look at Request Method/API section of developer documentation
- [ ] Facing difficulties? Reference to this issue and create a topic in [developer community](https://community.developers.freshworks.com/)

**Comment by mani-rsg** [comment]
Can you assign this to me ? @Saif-Shines

**Comment by ravirajsubramanian** [comment]
This issue is fixed in the PR #218.
